src/network/webtask.cpp

```diff
@@ -214,11 +214,11 @@ void WebTask::slotStart(int timeoutMillis, int delayMillis) {
             pNetworkAccessManager,
             timeoutMillis);
     if (!m_pendingNetworkReplyWeakPtr) {
-        kLogger.debug()
-                << this
-                << "Network request has not been started";
-        m_state = State::Aborted;
-        emitAborted(/*request URL is unknown*/);
+        m_state = State::Failed;
```

Comment:
The request cannot fail if it is never started. This behavior is not intended and is inconsistent.

Clients are responsible for checking the URL beforehand. Otherwise the request should fail as defined by the network manager.

Reply:
Adding opinionated code to the generic base classes instead of fixing the root cause in the derived classes is wrong.

Reply:
Sorry, it was wrong before and die to that the GUI was left in a stalled situation.
The function is named slotStart()  if it fails, the satus IS actually "Failed" and not artificialy abborded.
The point if a network API call has performed or not is an implementation detail.
Making here assumptions about the code in the calling class would be wrong.

Reply:
This case does not hapoen by the way. 
So the caller si fine .